ci-templates/.github/workflows/code-quality.yml

```diff
@@ -4,10 +4,12 @@ on:
   pull_request:
     branches:
       - production
+  types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
(applies throughout) I'm 99% certain this needs to be indented one more level under `pull_request`

Reply:
Ah, you're right, good catch. I'll fix it up.